includes/common/myparcel-constant.php

```diff
@@ -1,14 +1,11 @@
 <?php
 
-define('MYPARCEL_PLUGIN_VERSION', '2.1.14');
-define('SHIPMENT_PLUGIN_NOTICE', 'alive');
-define('GET_META_MYPARCEL_SHIPMENT_KEY', 'myparcel_shipment_key');
-define('GET_META_SHIPMENT_TRACKING_KEY', 'shipment_track_key');
-define('MYPARCEL_SUPPORT_TEXT_AND_URL', 'https://help.myparcel.com/home/integrations#Integrations-WooCommerce');
-define('MYPARCEL_API_SETTING_TEXT', 'MyParcel.com');
-define('MYPARCEL_WEBHOOK_URL', 'https://api.sandbox.myparcel.com/hooks');
-define('MYPARCEL_WEBHOOK_ACCESS_TOKEN', 'https://auth.sandbox.myparcel.com/access-token');
```

Comment:
FYI - this was for the hardcoded legacy webhook code with custom CURL calls, we now use our SDK